src/app.tsx

```diff
@@ -67,6 +69,12 @@ const AppContent = () => {
           <Route path="/claims" exact>
             <Claims />
           </Route>
+          <Route path="/policies/:policyId/claims/new" exact>
```

Comment:
Currently, the proposals are not created this way (the form opens in a dialog when you are on proposals page). I fear it will feel inconsistent.

Reply:
Is there any specific reason why not to include the claims to sidebar navigation?

Reply:
We don't yet have the icon, but I can probably add it for testing convenience so long